sdks/python/apache_beam/transforms/_external_transforms/__init__.py

```diff
@@ -0,0 +1,21 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+This is a subdirectory for generated external transforms.
+Refer to sdks/python/gen_xlang_wrappers.py for more details.
```

Comment:
As discussed offline, consider making storing the wrappers under transforms/xlang/... , and we can point readers of this file to generated docs that might have more info:

Consider following wording:

This is package has autogenerated Python wrappers for cross-language transforms available in other Beam SDKs .

For documentation on using the transforms, see: https://beam.apache.org/releases/pydoc/current/apache_beam.transforms.xlang.html

For more information on transform wrapper generation, see: sdks/python/gen_xlang_wrappers.py.